apps/editing-toolkit/editing-toolkit-plugin/global-styles/src/store.js

```diff
@@ -34,86 +37,69 @@ const actions = {
 	},
 };
 
-/**
```

Comment:
I removed this comment because it doesn't provide anything beyond what the type hints give us now.